ntbs-integration-tests/NotificationPages/TravelPageTests.cs

```diff
@@ -0,0 +1,459 @@
+            {
+                ["NotificationId"] = id.ToString(),
+                ["TravelDetails.HasTravel"] = "false",
+                ["VisitorDetails.HasVisitor"] = "false"
+            };
+
+            // Act
+            var result = await SendFormWithData(initialDocument, formData);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.Redirect, result.StatusCode);
+            Assert.Equal(BuildEditRoute(Routes.Comorbidities, id), GetRedirectLocation(result));
+        }
+
+        [Fact]
+        public async Task PostNotified_RedirectsToOverview_IfModelValid()
+        {
+            // Arrange
+            const int id = Utilities.NOTIFIED_ID;
+            var initialPage = await client.GetAsync(GetPageRouteForId(id));
+            var initialDocument = await GetDocumentAsync(initialPage);
+
+            var formData = new Dictionary<string, string>
+            {
+                ["NotificationId"] = id.ToString(),
+                ["TravelDetails.HasTravel"] = "false",
+                ["VisitorDetails.HasVisitor"] = "false"
+            };
+
+            // Act
+            var result = await SendFormWithData(initialDocument, formData);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.Redirect, result.StatusCode);
+            Assert.Equal(BuildRoute(Routes.Overview, id), GetRedirectLocation(result));
+        }
+
+        [Theory]
+        [InlineData(Utilities.DRAFT_ID)]
+        [InlineData(Utilities.NOTIFIED_ID)]
+        public async Task Post_ReturnsPageWithModelErrors_IfTotalNumberIsLessThanInputCountries(int id)
+        {
+            // Arrange
+            var initialPage = await client.GetAsync(GetPageRouteForId(id));
+            var initialDocument = await GetDocumentAsync(initialPage);
+
+            var formData = new Dictionary<string, string>
+            {
+                ["NotificationId"] = id.ToString(),
+                ["TravelDetails.HasTravel"] = "true",
+                ["TravelDetails.TotalNumberOfCountries"] = "1",
+                ["TravelDetails.Country1Id"] = "1",
+                ["TravelDetails.Country2Id"] = "2",
+
+                ["VisitorDetails.HasVisitor"] = "true",
+                ["VisitorDetails.TotalNumberOfCountries"] = "1",
+                ["VisitorDetails.Country1Id"] = "1",
+                ["VisitorDetails.Country2Id"] = "2",
+            };
+
+            // Act
+            var result = await SendFormWithData(initialDocument, formData);
+
+            // Assert
+            var resultDocument = await GetDocumentAsync(result);
+            result.EnsureSuccessStatusCode();
+
+            Assert.Equal(
+                FullErrorMessage(ValidationMessages.TravelOrVisitTotalNumberOfCountriesGreaterThanInputNumber),
+                resultDocument.QuerySelector("span[id='travel-total-error']").TextContent);
+
+            Assert.Equal(
+                FullErrorMessage(ValidationMessages.TravelOrVisitTotalNumberOfCountriesGreaterThanInputNumber),
```

Comment:
Can and will use @ChristianMcc 's document.GetError when it's merged in

Reply:
This is merged in now

Reply:
Yay, comms! :D